package.json

```diff
@@ -6,17 +6,16 @@
   "dependencies": {
     "express": "^4.14.0"
   },
+  "peerDependencies" : {
+    "bs-platform": "file:../bucklescript.git/"
+  },
   "devDependencies": {
-    "bs-platform": "^1.0.1"
+    "bs-platform": "file:../bucklescript.git/"
```

Comment:
As far as I understand, devDependencies don't really matter for library releases, unless you need a stable tag/branch with reproducible builds. When you install it with npm I think it'll just use whatever bs-platform is installed for the project the package is installed into.